test/spec/modules/adikteevBidAdapter_spec.js

```diff
@@ -20,21 +74,56 @@ describe('adikteevBidAdapter', () => {
     it('exists and is a function', () => {
       expect(adapter.callBids).to.exist.and.to.be.a('function');
     });
-    it('exists and is a function', () => {
-      expect(setstagingEnvironmentSwitch).to.exist.and.to.be.a('function');
-    });
-    it('exists and is correctly set', () => {
-      expect(stagingEnvironmentSwitch).to.exist.and.to.equal(false);
-    });
   });
 
+  describe('conformBidRequest', () => {
```

Comment:
Can you update your spec file to not use arrow functions. More info here https://github.com/prebid/Prebid.js/issues/2987

Reply:
Hey @jaiminpanchal27, done in https://github.com/prebid/Prebid.js/pull/3391/commits/542b03f04e51fdc57506a648a3ac5c7798e7ecd8 🙂 If these changes look good to you, would you mind approve these PR?

Reply:
@jaiminpanchal27 I believe this is patched, do you mind approving?